src/OSPSuite.UI/Views/Importer/ImporterDataView.cs

```diff
@@ -350,5 +350,10 @@ public void SetTabMarks(Cache<string, TabMarkInfo> tabMarks)
          refreshErrorMessage();
          refreshErrorMarks();
       }
+
+      public void ClearTabMarks()
+      {
+         SetTabMarks(new Cache<string, TabMarkInfo>(onMissingKey: _ => new TabMarkInfo(errorMessage: null, isLoaded: false)));
```

Comment:
the info on how the cache is created is leaking here. Probably the presenter should be the one to call SetTabMarks with the right init (or just clear the cache that it has locally presumably),. This can be merged but this could be optimized a bIT

Reply:
I think you are right... It will be enough with setting the marks again from the presenter.